roles/kubevirt-ssp/tasks/deprovision.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Check that kubevirt-ssp-operator.yml still exists in /tmp
+  stat:
```

Comment:
Why don't you use the https://docs.ansible.com/ansible/2.4/tempfile_module.html module?

Reply:
Is that because you want to avoid instantiating the template again when executed again?

Reply:
I was using the same convention as in other playbooks in the project, but I can change it accordingly

Reply:
Oh, lets keep it then, but I can't say I like this style..